scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1368,9 +1368,8 @@ class Router(formatOps: FormatOps) {
       // Type variance
       case tok @ FormatToken(T.Ident(_), T.Ident(_) | T.Underscore(), _)
           if isTypeVariant(leftOwner) =>
-        Seq(
-          Split(NoSplit, 0)
-        )
+        if (isSymbolicIdent(tok.right)) Seq(Split(Space, 0))
```

Comment:
Optional: i created a shortcut for this:

`Space(isSymbolicIdent(...))`...

Reply:
I forgot about it, thank you